src/azul/indexer/__init__.py

```diff
@@ -255,6 +266,26 @@ def __str__(self) -> str:
         """
         return f'{self.name}:{self.prefix}'
 
+    def contains(self, other: 'SourceSpec') -> bool:
+        """
+        >>> s = SimpleSourceSpec.parse('https://foo.edu:')
```

Comment:
Inline these variables but shorten the examples by extracting `p = SimpleSourceSpec.parse`. Newlines between cases as above.